{{ cookiecutter.package_name }}/.travis.yml

```
@@ -153,9 +153,13 @@ script:
     - tox $TOXARGS -- $TOXPOSARGS
 
 after_success:
-    # If coveralls.io is set up for this package, uncomment the two lines below.
-    # - pip install coveralls
-    # - coveralls
-    # If codecov is set up for this package, uncomment the two lines below
-    # - pip install codecov
-    # - codecov
+    # if either coveralls or codecov is used, uncomment the if and fi statements
+    # and the appropriate two lines inside the if statement
+    # - if [[ $TOXENV == *-cov ]]; then
+    #    # If coveralls.io is set up for this package, uncomment the two lines below.
+    #    pip install coveralls;
+    #    coveralls;
+    #    # If codecov is set up for this package, uncomment the two lines below
+    #    pip install codecov;
+    #    codecov;
+    #  fi
```

Is there a space missing here? Hard to tell.
```suggestion
    #   fi
```

---

Yep.  Good catch.